Library/OcCpuLib/OcCpuLib.c

```diff
@@ -874,7 +870,7 @@ ScanAmdProcessor (
 
   if (Cpu->Family == 0x0F) {
     switch (Cpu->ExtFamily) {
-      case 0x08:
+      case 0x08: //17h Family
```

Comment:
If you want to provide names to these numbers better make dedicated macros for them like `AMD_CPU_FAMILY_17H`. The comments formatted like this are invalid anyway, should be at least `///< 17h Family`, but better to just replace it with macros.

Reply:
thanks for spotting the mistakes i will correct it : )

Reply:
thanks for the suggestion will keep that in mind !